### ISSUE TITLE
No menu bar with Nanoboy 1.5

### ISSUE THREAD

**Issue by Felipefpl** [nba-emu/NanoBoyAdvance#215]
The menu bar isnt displayed here, it shows a black background, see below:

![No toolbar](https://user-images.githubusercontent.com/5569804/170896533-9b87ce91-a3c9-4d1a-89a4-713535d82d61.gif)

I have win7 x64 and intel hd 2000, i dont have any game boy emus or emus from other systems with this problem.

**Comment by fleroviux** [comment]
Unfortunately I can't reproduce this in my Windows 7 VM:
![image](https://user-images.githubusercontent.com/13669774/171471434-4a25cd6c-26cb-462f-9dd5-6f27d4317879.png)

Can you let me know what happens when you use Ctrl+O to open a ROM (you need to follow the process and assign a BIOS the first time)? Will it launch the game and output video at least?

**Comment by Felipefpl** [comment]
Added teh bios and tried pokemon emerald, the game loads, i can hear the music but i still get a black screen.

**Comment by fleroviux** [comment]
Hm. This might be the problem with OpenGL on older Intel integrated chips.
Though the way it shows (app still opens, without menu bar) is odd and not something I've seen before.

**Comment by Felipefpl** [comment]
I believe opengl 2 renderer isnt needed, i used aida64 and it showed i have 100% support for opengl 3.1 and 15% for opengl 4.0. Maybe the problem is something else. ;)

**Comment by fleroviux** [comment]
Your driver supports OpenGL 3.1, but not OpenGL 3.3 Core, which is what this emulator uses at the moment. 
The reason why I said OpenGL 2 fallback is because some older chips or drivers don't even support OpenGL 3.0, so I might as well handle all systems that don't support 3.3 Core through the same fallback.

**Comment by Felipefpl** [comment]
I see, indeed opengl 2 support seems a good idea. :+1:

**Comment by Felipefpl** [comment]
Tried 1.6 and the problem persists.

**Comment by Felipefpl** [comment]
Tested latest nightly 02/24/2023 and the screen is still black. :/

Btw, kongratz on your rewrite of PPU, i saw your work in the branch and surely it was something hard to do since it took you a lot of time to do it.

**Comment by fleroviux** [comment]
Yes, I have not got around to implement rudimentary support for older GL versions yet. I'll update you on this issue once I do, there is no need to test before that.

**Comment by Felipefpl** [comment]
> Yes, I have not got around to implement rudimentary support for older GL versions yet. I'll update you on this issue once I do, there is no need to test before that.

I see, good to know. ;)
